pkg/mapfs/fs.go

```diff
@@ -245,3 +245,7 @@ func cleanPath(path string) string {
 	path = strings.TrimLeft(path, "/") // Remove the leading slash
 	return path
 }
+
+func (m *FS) isPathAboveRoot(name string) bool {
+	return strings.HasPrefix(name, "..") && m.underlyingRoot != ""
```

Comment:
```suggestion
	return name == ".." || strings.HasPrefix(name, "../") && m.underlyingRoot != ""
```

A regular file can have `..` prefix.

```
$ touch ..foo
$ ls -al
total 0
drwxr-xr-x  3 teppei   96  9 25 11:06 ./
drwxrwxrwt 37 root   1184  9 25 11:06 ../
-rw-r--r--  1 teppei    0  9 25 11:06 ..foo
```

Reply:
Fixed https://github.com/aquasecurity/trivy/pull/7575/commits/a0023203081dbf101ac4a0d48f7ee570f3018e2d